src/UI.ts

```diff
@@ -100,6 +100,26 @@ export enum SelectionState {
   selected = "selected"
 }
 
+/** CSS classes used for animating months and captions. */
+export enum Animation {
+  /** Applied to the entering month when it is after the exiting month. */
+  animation_enter_month_weeks_is_after = "animation_enter_month_weeks_is_after",
```

Comment:
@gpbl FYI, I changed the name of these classes to `month_weeks` to follow the same patter of the `month_caption` and be more explicit that it's being applied to the `Weeks` component.